<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meat4Eat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/application.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
  background: transparent;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutMain">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetMain" native="true">
        <property name="styleSheet">
         <string notr="true">#widgetMain {
 border: 1px solid rgb(28,52,62);
 background-color: white;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,2">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetHead" native="true">
             <property name="cursor">
              <cursorShape>SizeAllCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#widgetHead {
  background-color: rgb(28,52,62);
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="labelLogo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>90</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>90</width>
                  <height>90</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="toolTip">
                 <string>Click to Visit the Meet4Eat Site</string>
                </property>
                <property name="pixmap">
                 <pixmap resource="../resources/application.qrc">:/logo.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelLogoText">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Roboto Black</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="toolTip">
                 <string>Click to Visit the Meet4Eat Site</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel#labelLogoText {
  color: rgb(231, 247, 167);
}</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;meet your friends&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;and have a nice time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="margin">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <layout class="QHBoxLayout" name="topBar">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonMimimize">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  padding: 0;
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-minimize.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonMaximize">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  padding: 0;
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-maximize.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonClose">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  padding: 0;
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-close.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                     <spacer name="verticalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLabel" name="labelStatus">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>8</pointsize>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QLabel#labelStatus {
  background-color: rgba(234, 234, 234, 0.6);
  border: 1px solid lightgray;
  border-radius: 4px;
}</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                      <property name="wordWrap">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QWidget" name="widgetToolbar" native="true">
               <property name="styleSheet">
                <string notr="true">#widgetToolbar {
  background-color: rgb(28,52,62);
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="spacing">
                   <number>12</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pushButtonEvents">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="toolTip">
                     <string>My events</string>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  background: transparent;
  border-image: url(:/icon-group.png);
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonSettings">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="toolTip">
                     <string>Settings</string>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  background: transparent;
  border-image: url(:/icon-settings.png);
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>2</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>2</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel {
  background-color: rgb(151,167, 187);
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonAbout">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>24</width>
                      <height>24</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>24</width>
                      <height>24</height>
                     </size>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="toolTip">
                     <string>About</string>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  background: transparent;
  border-image: url(:/icon-info.png);
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="verticalWidget" native="true">
               <property name="styleSheet">
                <string notr="true">#verticalWidget {
  background-color: rgb(28,52,62);
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,1">
                <item>
                 <widget class="QWidget" name="widgetSubMenu" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#widgetSubMenu {
  background-color: rgb(28,52,62);
}</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <property name="sizeConstraint">
                    <enum>QLayout::SetMinAndMaxSize</enum>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_4">
                     <property name="spacing">
                      <number>3</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widgetClientArea" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonEvents</tabstop>
  <tabstop>pushButtonSettings</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>pushButtonMimimize</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMimimize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnMinimizeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEvents</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnEventsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSettings</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAbout</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnAboutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMaximize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnMaximizeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnCloseClicked()</slot>
  <slot>onBtnMinimizeClicked()</slot>
  <slot>onBtnResizeReleased()</slot>
  <slot>onBtnEventsClicked()</slot>
  <slot>onBtnEventMgrClicked()</slot>
  <slot>onBtnSettingsClicked()</slot>
  <slot>onBtnAboutClicked()</slot>
  <slot>onBtnMaximizeClicked()</slot>
 </slots>
</ui>
